discovery-provider/src/queries/queries.py

```diff
@@ -131,6 +131,46 @@ def get_tracks():
     return api_helpers.success_response(tracks)
 
 
+# Get all tracks matching a route_id and track_id.
+# Expects a JSON body of shape:
+#   { "tracks": [{ "id": number, "url_title": string, "handle": string }]}
+@bp.route("/tracksIncludingUnlisted", methods=("POST",))
+def get_tracks_including_unlisted():
+    def validateIdentifiers (identifiers):
```

Comment:
regarding 'validateIdentifiers', should also be switched to snake case - why make this internal to this route? it might be useful for more functions later no?

Reply:
will switch to snake 🤦‍♂️

It's pretty tightly coupled to this route since it checks for the existence of this route's particular arguments. I could make it more generic and allow you to pass in arbitrary args to validate though...